gui/default/syncthing/core/syncthingController.js

```diff
@@ -1509,6 +1553,7 @@ angular.module('syncthing.core')
             }
 
             if (!done) {
+                deviceCfg.remoteGUIPort = "0"
```

Comment:
If this was a number, you wouldn't need to set this.

Reply:
I am not sure I understand. My thought process for adding this line is that I wanted to make sure that "0" was set on default (0 being the agreed upon signal to not show any link). remoteGUIPort is a string, so the default value is "".

Reply:
This is still an issue.